### ISSUE TITLE
auth response trigger failed

### ISSUE THREAD

**Issue by orion-belt** [hhorai/gnbsim#42]
I am testing gnbsim with oai-amf. I see with latest code (12 feb commit), UE auth is not getting triggered.

```
root@11c3255907ac:/gnbsim/example# 
root@11c3255907ac:/gnbsim/example# ./example 
[gnbsim]2021/02/13 20:12:02.245354 example.go:46: Dail LocalAddr: 127.0.0.1/192.168.62.198/192.168.61.198:48784; RemoteAddr: 192.168.61.195:38412
[gnbsim]2021/02/13 20:12:02.245503 example.go:66: write: len 44, info: &{Stream:0 SSN:0 Flags:0 _:0 PPID:1006632960 Context:0 TTL:0 TSN:0 CumTSN:0 AssocID:0}
[gnbsim]2021/02/13 20:12:02.247738 example.go:87: read: len 512, info: &{Stream:0 SSN:0 Flags:0 _:0 PPID:1006632960 Context:0 TTL:0 TSN:2206102550 CumTSN:0 AssocID:43}
dump: 2015003c0000040001000903004f41492d414d46005640011e0060000f010002f8595f00400064f0110b0041005000100002f859000116f000007b100800000c00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
Procedure Code: id-NGSetup (21)
PDU Length: 60
Protocol IEs: 4 items
  Item 0
    Protocol IE: id-AMFName (1)
      IE length: 9
        decoding id(1) not supported yet.
        dump: 03004f41492d414d46
  Item 1
    Protocol IE: id-RelativeAMFCapacity (86)
      IE length: 1
        decoding id(86) not supported yet.
        dump: 1e
  Item 2
    Protocol IE: id-ServedGUAMIList (96)
      IE length: 15
        decoding id(96) not supported yet.
        dump: 010002f8595f00400064f0110b0041
  Item 3
    Protocol IE: id-PLMNSupportList (80)
      IE length: 16
        decoding id(80) not supported yet.
        dump: 0002f859000116f000007b100800000c
[gnbsim]2021/02/13 20:12:02.248027 example.go:66: write: len 75, info: &{Stream:0 SSN:0 Flags:0 _:0 PPID:1006632960 Context:0 TTL:0 TSN:2206102550 CumTSN:0 AssocID:43}
[gnbsim]2021/02/13 20:12:02.251205 example.go:87: read: len 528, info: &{Stream:0 SSN:1 Flags:0 _:0 PPID:1006632960 Context:0 TTL:0 TSN:2206102551 CumTSN:0 AssocID:43}
dump: 0004403e000003000a000200020055000200010026002b2a7e005601020000218e940e783c954331ccb57e19731dcd78201022ec5414d4ce900114b8af90632592ecef0080000000a04000f48a7f00000000ef008b7f00000000000000000000d00bef008b7f0000a04000f48a7f0000ffffffffffffffff00000000000000000000000000000000203600e88a7f0000ffffffff8b7f0000700bef008b7f0000500bef008b7f0000a0af57088b7f00000000000000000000500def008b7f0000500cef008b7f0000a04000f48a7f0000300cef008b7f0000100cef008b7f0000600b00f48a7f0000d00bef008b7f0000000000000000000036997283c2550000020000000000000000e9fd912ef2bbaa400bef008b7f0000400bef008b7f0000500cef008b7f0000700def008b7f0000400cef008b7f000059c37283c2550000393500008a7f00000000000000000000703400e88a7f0000903400e88a7f00002800000030000000500cef008b7f0000800bef008b7f000000e9fd912ef2bbaa301800e88a7f00000000000000000000c00d00e88a7f0000180e00e88a7f00005f0000000000000000000000000000000000000000000000984200e88a7f0000984200e88a7f00000000000000000000707dbd83c2550000103a00e88a7f0000583a00e88a7f000010e502e08a7f0000903c00e88a7f0000e03b00e88a7f0000a04200e88a7f00000700000000000000
Procedure Code: id-DownlinkNASTransport (4)
PDU Length: 62
Protocol IEs: 3 items
  Item 0
    Protocol IE: id-AMF-UE-NGAP-ID (10)
      IE length: 2
  Item 1
    Protocol IE: id-RAN-UE-NGAP-ID (85)
      IE length: 2
cannot find camper for RanId=1
  Item 2
    Protocol IE: id-NAS-PDU (38)
      IE length: 43
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x54a626]

goroutine 34 [running]:
github.com/hhorai/gnbsim/encoding/ngap.(*GNB).SendtoUE(0xc0000c0400, 0x0, 0xc000050d20)
	/goproject/pkg/mod/github.com/hhorai/gnbsim/encoding/ngap@v0.0.0-20210213132930-3053d4e71c63/ngap.go:192 +0x36
github.com/hhorai/gnbsim/encoding/ngap.(*GNB).decNASPDU(0xc0000c0400, 0x0, 0xc000050f88, 0xc000050db0, 0x1)
	/goproject/pkg/mod/github.com/hhorai/gnbsim/encoding/ngap@v0.0.0-20210213132930-3053d4e71c63/ngap.go:1621 +0xdb
github.com/hhorai/gnbsim/encoding/ngap.(*GNB).decProtocolIE(0xc0000c0400, 0x0, 0xc000050f88, 0xc000050e38, 0x1, 0x1)
	/goproject/pkg/mod/github.com/hhorai/gnbsim/encoding/ngap@v0.0.0-20210213132930-3053d4e71c63/ngap.go:734 +0x4b5
github.com/hhorai/gnbsim/encoding/ngap.(*GNB).decProtocolIEContainer(0xc0000c0400, 0x0, 0xc000050f88, 0xc000050ed0, 0x1, 0x1)
	/goproject/pkg/mod/github.com/hhorai/gnbsim/encoding/ngap@v0.0.0-20210213132930-3053d4e71c63/ngap.go:669 +0x1e4
github.com/hhorai/gnbsim/encoding/ngap.(*GNB).Decode(0xc0000c0400, 0xc000050f88)
	/goproject/pkg/mod/github.com/hhorai/gnbsim/encoding/ngap@v0.0.0-20210213132930-3053d4e71c63/ngap.go:218 +0x205
main.(*testSession).recvfromAMF.func1(0xc0000a6200, 0xc000146000)
	/gnbsim/example/example.go:91 +0x28b
created by main.(*testSession).recvfromAMF
	/gnbsim/example/example.go:79 +0x7c
root@11c3255907ac:/gnbsim/example# 
```
[gnb_session.txt](https://github.com/hhorai/gnbsim/files/5976500/gnb_session.txt)

[gnbsim_pcap.zip](https://github.com/hhorai/gnbsim/files/5976505/gnbsim_pcap.zip)

**Comment by hhorai** [comment]
I have fixed the problem in d1e58dc.

**Comment by orion-belt** [comment]
Thanks @hhorai.
It worked for me. In Auth response message the criticality of IE UserLocationInformation is reject which should be ignore.
Isn't it ?

```
Item 3: id-UserLocationInformation
    ProtocolIE-Field
        id: id-UserLocationInformation (121)
        criticality: reject (0)
        value
            UserLocationInformation: userLocationInformationNR (1)
                userLocationInformationNR
                    nR-CGI
                    tAI
```

**Comment by hhorai** [comment]
Thank you very much for your feedback.
As you say, It is certainly a bug for gnbsim.
I've fixed the problem in 8c0ced7.

**Comment by hhorai** [comment]
Thank you for your advice. I have fixed it again in 31c2f40.
